sentry_sdk/integrations/redis.py

```diff
@@ -34,6 +55,11 @@ def setup_once():
             patch_redis_client(rb.clients.MappingClient)
             patch_redis_client(rb.clients.RoutingClient)
 
+        try:
+            _patch_rediscluster()
+        except Exception:
+            logger.exception("Error occured while patching `rediscluster` library")
```

Comment:
Not sure where this message will end up if we use `exception`, what about doing `logger.warning()` here?

Reply:
That should be fine. We get the log message attached even with exception